plugins/binanceExchange_ws_test.go

```diff
@@ -48,3 +48,36 @@ func Test_binanceExchangeWs_GetTickerPrice(t *testing.T) {
 		return
 	}
 }
+
+func Test_binanceExchangeWs_GetOrderBook(t *testing.T) {
+
+	testBinanceExchangeWs, e := makeBinanceWs()
+	if !assert.NoError(t, e) {
+		return
+	}
+
+	for _, obDepth := range []int32{1, 5, 8, 10, 15, 16, 20} {
+
+		pair := model.TradingPair{Base: model.XLM, Quote: model.BTC}
+		ob, e := testBinanceExchangeWs.GetOrderBook(&pair, obDepth)
+		if !assert.NoError(t, e) {
+			return
+		}
+		assert.Equal(t, ob.Pair(), &pair)
+
+		if !assert.True(t, len(ob.Asks()) > 0, len(ob.Asks())) {
```

Comment:
can you add a second check here to ensure that `len(ob.Asks()) <= obDepth`?
same for bids